.circleci/config.yml

```diff
@@ -0,0 +1,98 @@
+version: 2.1
+jobs:
+  build:
+    docker:
+      - image: amd64/centos:7
+    environment:
+      CC: /usr/bin/clang
+      BASH_ENV: /etc/profile.d/nextspace.sh
```

Comment:
@trunkmaster I noticed you made some new changes on your branch. I combined your changes with these new changes (these lines exist so that we don't need to set environment variables multiple times)

Reply:
@enzuru Actually this branch created by CircleCI. I've made some tests to get familiar with it. It seems I almost understand what's going on with this CI. Looks nice to me.

Since you've already merged my test config.yml into this PR I can remove CircleCI branch and continue to make changes to master.